examples/prometheus-operator/example.cue

```diff
@@ -0,0 +1,98 @@
+package prometheusOperator
+
+import "github.com/pollypkg/polly/schema:pollyschema"
+
+// NOTE we could make it an expectation that the emit value is
+// a pollyschema.PollyPackage
+prometheusOperator: pollyschema.PollyPackage & {
+	header: {
+		name: "prometheus-operator"
+		uri:  "github.com/pollypkg/polly/examples/prometheus-operator"
+		params: {}
+	}
+	signals: [
+		// List Errors
+		{
+			name: "ListErrors"
+			lang: "promql"
+			params: {job: string}
+			query: """
+                sum by (controller,namespace) (rate(prometheus_operator_list_operations_failed_total{job="\(params.job)"}[10m]))
+                /
+                sum by (controller,namespace) (rate(prometheus_operator_list_operations_total{job="\(params.job)"}[10m]))
+                > 0.4
+            """
+		},
+		// WatchErrors
```

Comment:
I did a shitty job of this in the other example, but i'd say we should imagine these comments to be like comments on a function in Go

By which i mean...more explanation :) A naive person (like me) should be able to read the comment and generally understand what i'm getting, here